Source/WebCore/Modules/highlight/HighlightRegister.h

```diff
@@ -55,10 +55,12 @@ class HighlightRegister : public RefCounted<HighlightRegister> {
     
     WEBCORE_EXPORT void addAnnotationHighlightWithRange(Ref<StaticRange>&&);
     const HashMap<AtomString, Ref<Highlight>>& map() const { return m_map; }
+    const ListHashSet<AtomString>& set() const { return m_set; }
     
 private:
     HighlightRegister() = default;
     HashMap<AtomString, Ref<Highlight>> m_map;
+    ListHashSet<AtomString> m_set; // holds highlight names in order
```

Comment:
Nit - perhaps, this could be named something like `m_highlightNamesInOrder` or just `m_orderedHighlightNames`? (And then we could remove the comment).

We could also just use a `Vector` here, since we already have `m_map` for fast `contains()` checks.